qa/workunits/rbd/concurrent.sh

```diff
@@ -88,9 +88,15 @@ function setup() {
 
 	# Set up some environment for normal teuthology test setup.
 	# This really should not be necessary but I found it was.
-	TOP="/tmp/cephtest"
+
+        if [ -f /etc/ceph/ceph.conf ]; then
+            TOP="/etc/ceph"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/ceph.client.0.keyring"
+        else
+            TOP="/tmp/cephtest"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
+        fi
 	export CEPH_ARGS="--conf ${TOP}/ceph.conf"
-	export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
```

Comment:
I fail to see how the old script overwrites conf:
        TOP="/tmp/cephtest"
        export CEPH_ARGS="--conf ${TOP}/ceph.conf"
        export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
        export CEPH_ARGS="${CEPH_ARGS} --name client.0"
So CEPH_ARGS would have "--conf FOO --keyring BAR --name BAZ".

Anyway, I think --conf and --keyring are handled by teuthology so I agree with Josh - --name should be sufficient.  (Actually I'm not even sure we need --name.  That script has more similar cruft in it: --secret to rbd map, udev dance, etc so dropping --conf and --keyring entirely should be a step in the right direction.)